docs/guided-tour/blueprints/helm-chart-resource/README.md

```diff
@@ -1,81 +1,114 @@
-# Helm Chart Resources in the Component Descriptor
+# Helm Chart Resources in the Component
```

Comment:
What about referencing a helm chart in a helm chart repository? This is currently our most important customer use case.

Reply:
Added an example of a corresponding component configuration file